app/src/main/java/com/firebase/ecommerce/feature_login/data/repository/RegistrationRepositoryImp.kt

```diff
@@ -73,6 +74,24 @@ class RegistrationRepositoryImp @Inject constructor(
         }
     }
 
+    override fun initiatePasswordReset(email: String): Flow<Resource<AuthResult>> {
+        return flow {
+            firebaseAuth.sendPasswordResetEmail(email)
+                .addOnCompleteListener { task ->
+                    if (task.isSuccessful) {
+                        // Password reset email sent successfully
+                        // Show a success message or navigate to a confirmation screen
```

Comment:
@Ankita123-oss I didn't see any password reset logic here, is it missed or included in some other code?